src/actions/Upload.ts

```diff
@@ -86,6 +90,26 @@ export class Upload extends Edit {
     return Promise.all(uploads)
   }
 
+  private async uploadDeobfuscation () {
+    debug('> Uploading deobfuscation')
+    if (this.uploadParams.deobfuscation) {
+      return this.publisher.edits.deobfuscationfiles.upload(
+        {
+          apkVersionCode: this.editParams.versionCode,
+          deobfuscationFileType: 'proguard',
+          editId: this.editId,
+          media: {
+            body: createReadStream(this.uploadParams.deobfuscation),
+            mimeType: 'application/octet-stream'
+          },
+          packageName: this.editParams.packageName
+        },
+          {}
```

Comment:
Slight code-style issue 🙂 
```suggestion
        {}
```